.github/workflows/build_cmake_macos.yml

```diff
@@ -0,0 +1,112 @@
+---
+name: Main test matrix for macOS
+
+on:
+  push:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - 'doc/**'
+      - 'CHANGES'
+      - 'COPYING'
+      - 'CREDITS'
+      - 'LICENSE'
+      - 'README.*'
+
+env:
+  POCL_CACHE_DIR: "/tmp/GH_POCL_CACHE"
+  CTEST_FLAGS: "--output-on-failure --test-output-size-failed 128000 --test-output-size-passed 128000"
+  POCL_KERNEL_CACHE: "0"
+  POCL_MAX_WORK_GROUP_SIZE: "1024"
+  POCL_MEMORY_LIMIT: "2"
+
+jobs:
+  main_test_matrix:
+    name: LLVM macOS ${{ matrix.llvm }} - ${{ matrix.config }}
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        llvm: [15, 16]
+        # basic = simplest pthread build
+        # devel = with devel options
+        # cts_spirv = cts with SPIR-V
+        # tier1 = includes CTS without SPIR-V
+        # asan, tsan, ubsan = sanitizers
+        config: [basic]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: conda-incubator/setup-miniconda@v2
```

Comment:
This needs to be added as an allowed action. It's used for installing conda to install llvm, clang, etc

Reply:
added